app/views/standard/upload.scala.html

```diff
@@ -48,60 +50,53 @@ <h2 class="govuk-error-summary__title" id="error-summary-title">
-                                </div>
-                            </dd>
+                    <div class="govuk-form-group">
+                        <div id="selection-area">
+                            @itemSelectionFailureMessage("multiple-folder-selection-failure", "multiple-folder-selected-message-text", "upload.dragAndDropMultipleFoldersErrorMessage")
+                            @itemSelectionFailureMessage("multiple-selection-failure", "multiple-object-type-selected-message-text", "upload.standard.dragAndDropMultipleFilesErrorMessage")
+                            @itemSelectionFailureMessage("item-selection-failure", "wrong-object-type-selected-message-text", "upload.dragAndDropFolderErrorMessage")
+                            @itemSelectionFailureMessage("nothing-selected-submission-message", "submission-without-anything-selected-text", "upload.submissionWithoutAFolderMessage")
+                            <div class="drag-and-drop__dropzone">
+                                <input type="file" id="file-selection" name="files"
+                                class="govuk-file-upload drag-and-drop__input" webkitdirectory
+                                @* Specify an arbitrary type in the 'accept' attribute to work around a bug in
+                                Safari 14.0.1, which does not let the user browse for files if the 'accept'
+                                attribute is missing. The actual value of the attribute is ignored because
+                                'webkitdirectory' is specified. It just needs to be present to fix the Safari bug. *@
+                                accept="*"
+                                >
+                                <p class="govuk-body drag-and-drop__hint-text">Drag and drop a single top-level folder here or &nbsp;</p>
+                                <label for="file-selection" class="govuk-button govuk-button--secondary drag-and-drop__button">
+                                    Choose folder
+                                </label>
+                            </div>
                         </div>
-                    </div>
-                    <div>
+                        <div id="success-and-removal-message-container" class="govuk-summary-list govuk-file-upload" hidden="true">
+                            <div class="js-drag-and-drop-selected drag-and-drop__selected" id="item-selection-success-container">
+                                <p id="success-message-text" aria-live="assertive" aria-atomic="true" class="govuk-!-margin-bottom-3 govuk-!-margin-top-0 drag-and-drop__selected__description">The folder <strong id="files-selected-folder-name" class="folder-name"></strong> (containing <span class="folder-size"></span> files) has been selected.</p>
```

Comment:
Can remove the word "files" from here (containing <span class="folder-size"></span> files) as it is duplicated in the success message

Reply:
Good catch. Removed it, thanks.